notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import cv2
</pre><p>This import requires opencv-python to be installed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
added installation of the require package